### PR TITLE
Fix to_dict methods for stressmodels

### PR DESCRIPTION
**Pull request by raoulcollenteur**
# Short Description

Many of the to_dict methods for the different stressmodels were missing or wrong. I added the method everywhere and ensured that the exported keywords and values match those of the _init__ of each method. This ensures the stressmodels are the same when doing ml.copy or exporting/loading pas-files.

# Checklist before PR can be merged:
- [x] closes issue #486
- [x] is documented
- [x] PEP8 compliant code
- [x] tests added / passed

### CODE REVIEW

**Comment by raoulcollenteur** [comment]
This PR is ready for review.

File: pastas/stressmodels.py [hunk 1]
```diff
@@ -1678,6 +1731,29 @@ def simulate(
 
         return h
 
+    def to_dict(self, series: Optional[bool] = True):
```
Comment (martinvonk) [inline pastas/stressmodels.py:1734]: No `Optional[bool]` but `bool`.

`Optional` = `Union[type, None]` = `type | None`

**Review by martinvonk — APPROVED** [review]

File: pastas/stressmodels.py (hunk 1 above)
Comment (raoulcollenteur) [reply pastas/stressmodels.py:1734]: Thanks for the type hint. I'll change it. I thought this is that now this argument is optional but I guess not.

**Comment by raoulcollenteur** [comment]
Thanks for the review @martinvonk ! I'll merge this as soon as GH Actions is done and green.

**Pull request merged**